README.md

```diff
@@ -141,6 +141,60 @@ datafusion = "6.0.0"
 
 DataFusion also includes a simple command-line interactive SQL utility. See the [CLI reference](https://arrow.apache.org/datafusion/cli/index.html) for more information.
 
+# Roadmap
+
+A quarterly roadmap will be published to give the DataFusion community visibility into the priorities of the projects contributors. This roadmap is not binding.
+
+## 2022 Q1
+
+### DataFusion Core
+
+- Publish official Arrow2 branch
```

Comment:
Hi @alamb! I'm curious what "arrow2" means here. Is it related to https://github.com/jorgecarleitao/arrow2?

Reply:
Yes that's what it refers to

Reply:
yes, that is exactly what arrow2 means

There is a branch https://github.com/apache/arrow-datafusion/tree/arrow2 and a discussion ticket https://github.com/apache/arrow-datafusion/issues/1532 that has more information if you are interested

Reply:
Thank you!